ios/Runner/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10117" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="BYZ-38-t0r">
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="10085"/>
+    </dependencies>
+    <scenes>
+        <!--Flutter View Controller-->
+        <scene sceneID="tne-QT-ifu">
+            <objects>
+                <viewController id="BYZ-38-t0r" customClass="FlutterViewController" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="y3c-jy-aDJ"/>
+                        <viewControllerLayoutGuide type="bottom" id="wfy-db-euE"/>
```

Comment:
ditto.  what is the difference of Main.storyboard and LaunchScreen.stroryboard?

Reply:
The Launch Screen serves as a brief introductory screen to an app, while the Main Storyboard defines the core user experience once the app is running.

Reply:
ditto

Reply:
**LaunchScreen.storyboard:** t’s the splash screen that we see right when we tap on the app icon and before the main content of our app loads. It’s usually a simple screen with our app's logo or a background image.

**Main.storyboard:** Main.storyboard is used to show the real content of the app once it’s up and running (in this case, the Flutter app). It’s used after the app has been launched, once the app is ready to show the main content created using Flutter.